uint64 timeStamp
uint16 nSignals
uint16 halfFrameSize
float32[] data
